components/portfolio/BalanceRow.vue

```diff
@@ -128,24 +121,15 @@ export default {
       }
       let colour = '#'
       for (let i = 0; i < 3; i++) {
-        // eslint-disable-next-line prettier/prettier
-        const value = (hash >> (i * 8)) & 0xFF
+        // eslint-disable-next-line
+        const value = (hash >> (i * 8)) & 0xff
```

Comment:
sure this works?

Reply:
just linting nonsense

i will revert and see if the linter doesn't complain

Reply:
my linter is forcing it to `0xff`